.github/workflows/_build_test_upload.yml

```diff
@@ -504,6 +504,7 @@ jobs:
         run: |
           cd ./docs
           pip3 install -r requirements.txt
+          make doctest
```

Comment:
I would rather put this test into `ci.yml`. This `_build_test_upload` is the one we upload compiled binaries.

All PR will execute GHA in `ci.yml`.

Reply:
I have put it into ci.yml now.